lib/static/new-ui/components/AssertViewResult/index.module.css

```diff
@@ -0,0 +1,17 @@
+.diff-viewer-container {
+    display: flex;
+    flex-direction: column;
+    padding-left: calc(var(--indent) * 24px);
+    padding-right: 1px
+}
+
+.diff-mode-switcher {
+    --g-color-base-background: #fff;
+    margin: 12px auto;
+}
+
+.screenshot {
+    margin: 8px 0;
+    padding-left: calc(var(--indent) * 24px);
+    padding-right: 1px;
```

Comment:
can't understand why we need this indentation here and in the `diff-viewer-container` block

Reply:
Because screenshots are nested in steps tree, namely inside assertView command which can be inside some other command, e.g. runStep. These paddings are to maintain correct indents.